### PR TITLE
Fixed CSS for image rendering small

### LINKED ISSUE #460: embedded image is tiny
See, for example, https://ocwnext.odl.mit.edu/courses/14-01sc-principles-of-microeconomics-fall-2011/pages/unit-1-supply-and-demand/introduction-to-microeconomics/

### PR DESCRIPTION
**Pull request by Wassaf-Shahzad**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
- [x] Testing
   - [x] Changes have been manually tested

#### What are the relevant tickets?
Closes #460 

#### What's this PR do?
Fixes css to images expands to original width

#### How should this be manually tested?
Clone Course with embedded image
- Verify image is rendered correctly 

#### Screenshots (if appropriate)
Screenshot 2022-02-23 at 3 03 31 PM

### CODE REVIEW

**Comment by Wassaf-Shahzad** [comment]
FYI,
The image in the screenshot is Open Learning since the resource file was not rendering due to access issue.

File: base-theme/assets/css/main.scss [hunk 1]
```diff
@@ -299,6 +299,13 @@ table {
   img {
     max-width: 100%;
   }
+  td {
+    img {
+      // Incase of img in td setting maxwith to 100% will lead to img
+      // to only have the width of the table column
+      max-width: none;
```
Comment (MAbdurrehman12) [inline base-theme/assets/css/main.scss:306]: Just to be on a safer side, do we need to increase it's priority by adding `!important`?

**Review by MAbdurrehman12 — APPROVED** [review]
👍

**Comment by MAbdurrehman12** [comment]
> FYI, The image in the screenshot is Open Learning since the resource file was not rendering due to access issue.

If you parse the course in `ocw-to-hugo` and use the output result then all the images will render I guess.
image

image

**Pull request merged**